src/sentry/notifications/notifications/activity/new_processing_issues.py

```diff
@@ -18,11 +18,16 @@ def __init__(self, activity: Activity) -> None:
 
     def get_participants_with_group_subscription_reason(
         self,
-    ) -> Mapping[ExternalProviders, Mapping[User, int]]:
-        users_by_provider = NotificationSetting.objects.get_notification_recipients(self.project)
+    ) -> Mapping[ExternalProviders, Mapping[Team | User, int]]:
```

Comment:
I don't think this hurts anything, but I also don't think a team would ever be notified about processing issues.

Reply:
I just needed the method signature to match the parent class to satisfy the type checker.